pkgs/tools/security/truecrack/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, gccStdenv, fetchFromGitLab, cudatoolkit
```

Comment:
does it explicitly need gcc?

Reply:
Yes, it fails with clang. And on MacOS Clang is the default stdenv.cc

Reply:
sounds good then